### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by guillenvd**
- Updates Docker file to use previous working reference

### CODE REVIEW

**Comment by blakemotl-RL** [comment]
Even better would be to pin the sha hash of the image. Its unacceptable that even though we pinned the action hash to reduce supply chain risk, we were still affected by a change.

**Comment by blakemotl-RL** [comment]
Even better would be to pin the sha hash of the image. Its unacceptable that even though we pinned the action hash to reduce supply chain risk, we were still affected by a change.

**Comment by relrod** [comment]
Dupe of #67

**Comment by adrian-livefront** [comment]
Duplicate of #67

**Review by k1ch — APPROVED** [review]

**Comment by tovbinm** [comment]
Thanks! @guillenvd 

@k1ch can you please merge & release?

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM sonarsource/sonar-scanner-cli:5.0
+FROM sonarsource/sonar-scanner-cli:5.0.0
```
Comment (k1ch) [inline Dockerfile:1]:
```suggestion
FROM sonarsource/sonar-scanner-cli:5.0

USER root
```

**Comment by k1ch** [comment]
> Thanks! @guillenvd
> 
> @k1ch can you please merge & release?

Unfortunately I do not have access to do it.

**Review by sourceful-karlson — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (AntonyF-Andreani) [reply Dockerfile:1]: Teh versions apparently published are:

image

So I think this PR not have much sense

**Review by k1ch — COMMENTED** [review]
The issue with `sonarsource/sonar-scanner-cli:5.0` has already been resolved. This PR can now be closed.

**Pull request closed without merging**